var/spack/repos/builtin/packages/chameleon/package.py

```diff
@@ -113,15 +113,13 @@ def cmake_args(self):
                 ]
             )
 
-        if spec.satisfies("~simgrid"):
```

Comment:
why is this dropped?

Reply:
To simplify, since the package depends on lapack / blas only when `+simgrid`.

Reply:
but dependencies could